sourcegraph/sourcegraph
# insights: add ping for exporting data clicks

closes #47433 

analytics PR https://github.com/sourcegraph/analytics/pull/795

<img width="952" alt="Screenshot 2023-02-16 at 18 16 49" src="https://user-images.githubusercontent.com/29406849/219453355-29d9e91d-3faa-4fd0-9365-b83d0812cb2a.png">

## Test plan

Manually tested.
If any error, this will not break pings thanks to our lord and saviour @coury-clark. Adding pings has never been so easy 🙏 

Manual test plan (writing this down because I _always_ forget the second step):
* in your local site-config, set `disableNonCriticalTelemetry` to false
* update the[ client-check delay](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:%5Ecmd/frontend/internal/app/updatecheck/client%5C.go+ctx%2C+cancel+:%3D&patternType=standard&sm=1&groupBy=path) to something small
* navigate to your pings and profit
